components/studio/build-docker-image.ps1

```diff
@@ -1,97 +0,0 @@
-param ([String[]]$InstallHarts = $null)
```

Comment:
Did you mean to delete this file as part of this commit? It wasn't mentioned in the PR message.

Reply:
I did. Thanks for asking! It was used by the `bintray-publish` that was previously removed. I just added a separate commit for this deletion.